code/modules/mob/living/bot/farmbot.dm

```diff
@@ -348,7 +348,7 @@
 		user << "You complete the Farmbot! Beep boop."
 		var/mob/living/bot/farmbot/S = new /mob/living/bot/farmbot(get_turf(src))
 		for(var/obj/structure/reagent_dispensers/watertank/wTank in contents)
-			qdel(S.tank)
+			QDEL_NULL(S.tank)
```

Comment:
Why is this necessary? The loop sets a new reference already.

Reply:
*shrug emoji* so that it deletes before setting a new reference.